UNLV-CS472-672/2024-S-GROUP5-Munch
# Created Recipe Page UI

## Description / Changes Made

- Added Path to the Home Page "The Recipe" Button
- Created new recipe page
- added back button
- added recipe image
- added title
- added subtitle
- added body text
- added recipe page temporarily to the navigation tab for easy access for testing

## Screenshots (optional)


## How to Test

1. Load the app
2. click the "the recipe" button
3. see if it leads to the recipe page
4. click the "back" button 
5. see if it leads back to the home page
![Screenshot_20240326_162749](https://github.com/UNLV-CS472-672/2024-S-GROUP5-Munch/assets/136287986/4882cf5d-000f-4804-bd24-c9dd97ed5dd7)

## Checklist
